test/integration/taxon_browsing_test.rb

```diff
@@ -124,6 +126,14 @@ def and_i_can_see_the_services_section
     assert page.has_link?(expected_link[:text], href: expected_link[:url])
   end
 
+  def and_i_can_see_the_news_and_communications_section
+    assert page.has_content?('News and communications')
+
+    tagged_content.each do |item|
+      assert page.has_link?(item["title"], href: item["link"])
+    end
```

Comment:
Should probably add a check for the "See all ..." link similar to the services, and guidance and regulation sections.

Reply:
Whoops, will add that in..